package.json

```diff
@@ -0,0 +1,30 @@
+{
```

Comment:
Make sure you also use `package.lock` file. Thanks to this you will avoid problems with future versions of the dependencies 💪